### PR TITLE
added homepage

### PR DESCRIPTION
**Pull request by RohitKuwar**
@rajakvk Please review PR.

### CODE REVIEW

File: components/Clock.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      const formatedText = h + ':' + m + session + ' ' + 'IST'
+      this.timer = formatedText
+      setTimeout(this.clock, 1000)
+    },
+  },
+}
+</script>
+
+<style scoped>
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@600&display=swap');
```
Comment (SBagaria2710) [inline components/Clock.vue:39]: Why is this font scoped? Shouldn't it be available to other components in the application?

File: components/Clock.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  background-color: #e30062;
+  float: right;
+  box-shadow: -1px 3px 4px #1d1283;
+  display: block;
+  justify-content: center;
+}
+.date-time {
+  color: #ffffff;
+  font-family: 'Saira Semi Condensed', sans-serif;
+  font-size: 21px;
```
Comment (SBagaria2710) [inline components/Clock.vue:51]: Try to use size in multiple of 4, or 2 if you had to.

File: components/Days.vue [hunk 3]
```diff
@@ -0,0 +1,192 @@
+<template>
+  <div class="week-box">
+    <div class="heading">
+      <img src="~/assets/ankur.png" alt="userpic" />
+      <h1>Your goals</h1>
+    </div>
+    <h2 class="week-heading">This week</h2>
+    <div class="days">
+      <div class="day-box">
```
Comment (SBagaria2710) [inline components/Days.vue:9]: Could have maintained an array for the days with their completion status, and mapped it.
It shouldn't have been this verbose.
.
Just a suggestion.

File: components/Days.vue [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+</template>
+
+<script>
+export default {
+  name: 'Days',
+}
+</script>
+
+<style scoped>
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@800&display=swap');
```
Comment (SBagaria2710) [inline components/Days.vue:48]: Same font as been imported here as well. It was also imported in `Clock.vue`

File: store/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# STORE
```
Comment (SBagaria2710) [inline store/README.md:1]: You can, maybe, remove these files if they weren't required.
Also, you can maintain a `.gitkeep` file if you wish to keep these folders.

**Comment by SBagaria2710** [comment]
Hey @RohitKuwar,

I did a rough review. Will run it on local and provide more feedback :)
You can go through the comments, and please let me know if any further clarifications are required for any of the comments.

Overall, things are looking good tho 👍🏼

**Comment by RohitKuwar** [comment]
> Hey @RohitKuwar,
> 
> I did a rough review. Will run it on local and provide more feedback :)
> You can go through the comments, and please let me know if any further clarifications are required for any of the comments.
> 
> Overall, things are looking good tho 👍🏼

@SBagaria2710 Thank you for the comments. Will work in it.

File: store/README.md (hunk 5 above)
Comment (RohitKuwar) [reply store/README.md]: done

File: components/Days.vue (hunk 4 above)
Comment (RohitKuwar) [reply components/Days.vue]: done.

File: components/Days.vue (hunk 3 above)
Comment (RohitKuwar) [reply components/Days.vue]: done

File: components/Clock.vue (hunk 2 above)
Comment (RohitKuwar) [reply components/Clock.vue]: done

File: components/Clock.vue (hunk 1 above)
Comment (RohitKuwar) [reply components/Clock.vue]: done

File: components/Days.vue [hunk 6]
```diff
@@ -178,12 +172,17 @@ export default {
...
     border: 2px solid #025300;
   }
-  .inprogress {
+  .day-box:nth-child(5) button {
     width: 14px;
     height: 14px;
     border: 2px dashed #e6a500;
   }
-  .incomplete {
+  .day-box:nth-child(6) button {
```
Comment (SBagaria2710) [inline components/Days.vue:180]: Shouldn't these things be dynamic?
Is that something we'd be doing later?

File: layouts/default.vue [hunk 7]
```diff
@@ -5,6 +5,11 @@
 </template>
 
 <style>
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@800&display=swap');
```
Comment (SBagaria2710) [inline layouts/default.vue:10]: You have 2 fonts which are identical, only the `wght` differs which I believe implies for `font-weight`.
So, would it make any difference if you import any one and configure the `font-weight` in the CSS file itself?

Just curious

**Review by SBagaria2710 — COMMENTED** [review]
@RohitKuwar Have added a couple comments, could you please go through them?

File: layouts/default.vue (hunk 7 above)
Comment (RohitKuwar) [reply layouts/default.vue:10]: @SBagaria2710 Yes, you are right. We have same `font` with different `font-weight`. But the issue is, it is not working just by importing that particular `font` & giving different `font-weights`. If possible to you can you please look into it. May be I am missing something.

File: components/Days.vue (hunk 6 above)
Comment (SBagaria2710) [reply components/Days.vue:180]: Cool then

File: layouts/default.vue (hunk 7 above)
Comment (SBagaria2710) [reply layouts/default.vue:10]: That shouldn't be the case. I'll check it and will update you.
For now, approving it from my end.

**Comment by RohitKuwar** [comment]
> 
> 
> @RohitKuwar Have added a couple comments, could you please go through them?

@SBagaria2710 Ok

File: layouts/default.vue (hunk 7 above)
Comment (RohitKuwar) [reply layouts/default.vue:10]: > 
> 
> That shouldn't be the case. I'll check it and will update you.
> For now, approving it from my end.

OK thank you

**Pull request merged**